indexer/src/constants.ts

```diff
@@ -50,3 +50,11 @@ export const KAKAROT_ADDRESS: string = (() => {
   if (!kakarotAddress) throw new Error("ENV: KAKAROT_ADDRESS is not set");
   return kakarotAddress;
 })();
+
+// Path to the JSON file containing real transaction data
+export const TRANSACTIONS_DATA_FILE =
+  "indexer/src/test-data/transactionsData.json";
+
+// Path to the JSON file containing the expected data trasformed with transform function and toTypedEthTx
+export const EXPECTED_TRANSFORM_DATA_FILE =
+  "indexer/src/test-data/expectedTransformData.json";
```

Comment:
I think we should move the constants that are test related to another file.

Reply:
Makes sense, fixed. Thanks